.travis.yml

```diff
@@ -32,6 +32,13 @@ before_install:
 - if [ "$TRAVIS_OS_NAME" = "linux" ]; then USE_NINJA="-GNinja"; VALGRIND="valgrind $VALGRIND_OPTIONS"; . /opt/qt57/bin/qt57-env.sh; fi
 
 install:
+# olm
+- if [ "$TRAVIS_OS_NAME" = "osx" ]; then brew update; else sudo apt-get update -qq; fi
+- if [ "$TRAVIS_OS_NAME" = "osx" ]; then brew install qt5; export PATH="$PATH:/usr/local/opt/qt/bin"; else sudo apt-get install -y qt5-default; fi
+- git clone https://matrix.org/git/olm.git && cd olm && make && sudo make install && cd ..
```

Comment:
removed `sudo` part, but probably do need `olm` to build `qtolm`, so provided new part to install it to the build env :thinking:

Reply:
Looked at Travis CI and found that my current solution is searching for the non-existing `olm` system package (since it's not my native environment :see_no_evil: ) Going to provide additional MR to `libQtOlm` with "in-tree" olm build support and `olm` submodule.

Reply:
I'd say best is to provide a way for a user to point to his olm headers and libs, rather than embed it in-tree.